common/testData/localeIdentifiers/localeCanonicalization.txt

```diff
@@ -0,0 +1,1647 @@
+# Test data for locale identifier canonicalization
+#  Copyright © 1991-2020 Unicode, Inc.
+#  For terms of use, see http://www.unicode.org/copyright.html
+#  Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
+#  CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+#
+# Format:
+# <source locale identifier>	;	<expected canonicalized locale identifier>
+#
+# The data lines are divided into 4 sets:
+#   explicit: a short list of explicit test cases.
+#   fromAliases: test cases generated from the alias data.
+#   decanonicalized: test cases generated by reversing the normalization process.
+#   withIrrelevants: test cases generated from the others by adding irrelevant fields where possible,
+# to ensure that the canonicalization implementation is not sensitive to irrelevant fields. These include:
+#     Language: aaa
+#     Script:   Adlm
+#     Region:   AC
+#     Variant:  fonipa
+######
+
+
+
+# explicit
+
+art_lojban	;	jbo
+en_US_aaland	;	en_US
+en_US_heploc	;	en_US_alalc97
+en_US_polytoni	;	en_US_polyton
+en_aaland	;	en_AX
+en_arevela	;	en
+en_arevmda_arevela	;	en
+en_lojban	;	en
+hy_arevela	;	hy
+hy_arevmda	;	hyw
+hy_arevmda_arevela	;	hyw
+hye_arevmda	;	hyw
+no_bokmal_nynorsk	;	nb
+no_nynorsk_bokmal	;	nb
+zh_guoyu_hakka_xiang	;	hak
+zh_hakka_xiang	;	hak
+
+# fromAliases
+
+aa_saaho	;	ssy
+aam	;	aas
+aar	;	aa
+abk	;	ab
+adp	;	dz
+afr	;	af
+aju	;	jrb
+aka	;	ak
+alb	;	sq
+als	;	sq
+amh	;	am
+ara	;	ar
+arb	;	ar
+arg	;	an
+arm	;	hy
+asd	;	snz
+asm	;	as
+aue	;	ktz
+ava	;	av
+ave	;	ae
+aym	;	ay
+ayr	;	ay
+ayx	;	nun
+aze	;	az
+azj	;	az
+bak	;	ba
+bam	;	bm
+baq	;	eu
+bcc	;	bal
+bcl	;	bik
+bel	;	be
+ben	;	bn
+bgm	;	bcg
+bh	;	bho
+bih	;	bho
+bis	;	bi
+bjd	;	drl
+bod	;	bo
+bos	;	bs
+bre	;	br
+bul	;	bg
+bur	;	my
+bxk	;	luy
+bxr	;	bua
+cat	;	ca
+ccq	;	rki
+cel_gaulish	;	xtg
+ces	;	cs
+cha	;	ch
+che	;	ce
+chi	;	zh
+chu	;	cu
+chv	;	cv
+cjr	;	mom
+cka	;	cmr
+cld	;	syr
+cmk	;	xch
+cmn	;	zh
+cnr	;	sr_ME
```

Comment:
I believe this should be cnr ; sr_Latn_ME because we also have
    		<languageAlias type="sr_ME" replacement="sr_Latn_ME" reason="legacy"/>
in the data.

Reply:
sr_Latn_ME ~ sr_ME, since sr in ME is written in Latn.